certs/variables.tf

```diff
@@ -1,66 +1,81 @@
-# openvpn_server - Variables
+# Input Variables
 
-variable "ami" {
-  description = "Artifact AMI"
+## Resource tags
+variable "stack_item_label" {
+  type        = "string"
+  description = "Short form identifier for this stack. This value is used to create the 'Name' resource tag for resources created by this stack item, and also serves as a unique key for re-use."
 }
 
-variable "stack_item_fullname" {}
+variable "stack_item_fullname" {
+  type        = "string"
+  description = "Long form descriptive name for this stack item. This value is used to create the 'application' resource tag for resources created by this stack item."
+}
 
-variable "stack_item_label" {}
+## VPC parameters
 
-#
-# 0 - if instance is a standalone instance outside a VPC
-# 1 - if instance is in a vpc
-#
+###
+### 0 - if instance is a standalone instance outside a VPC
+### 1 - if instance is in a vpc
+###
 variable "in_vpc" {
-  default = 0
+  type        = "string"
+  description = "Flag for associating the cluster with a VPC."
+  default     = 1
 }
 
 variable "vpc_id" {
-  default = ""
+  type        = "string"
+  description = "ID of the target VPC."
 }
 
-variable "security_groups" {
-  default = ""
+variable "region" {
+  type        = "string"
+  description = "AWS region to be utilized."
 }
 
-# Which subnet the vpn servers will run in.
 variable "subnets" {
-  default = ""
+  type        = "string"
+  description = "List of VPC subnets to associate with the cluster."
 }
 
-# TODO: expects 4 subnets to map as internal network routes.
-# Fix the magic # problem
-variable "route_cidrs" {
-  default = ""
+## OpenVPN parameters
+variable "ami" {
+  type        = "string"
+  description = "Amazon Machine Image (AMI) to associate with the launch configuration."
 }
 
-variable "key_name" {}
-
-#
-# m3.medium - if instance is a standalone instance outside a VPC
-# t2.small - if instance is in a vpc
-#
-variable "instance_type" {}
-
-variable "region" {}
-
-variable "release" {
```

Comment:
Hrmm, yeah this probably isn't the best place to track it. Wanted to somehow attach packer version numbers to the terraform builds. Any ideas?